packages/api/src/mails/mailContext.ts

```diff
@@ -0,0 +1,125 @@
+import {BaseMailProvider} from './mailProvider'
+import {DBAdapter} from '../db/adapter'
+import {MailLogState} from '../db/mailLog'
+import {logger} from '../server'
+
+import Email from 'email-templates'
+
+export enum SendMailType {
+  LoginLink,
+  TestMail,
+  PasswordReset,
+  NewMemberSubscription,
+  RenewedMemberSubscription,
+  MemberSubscriptionOffSessionBefore,
+  MemberSubscriptionOnSessionBefore,
+  MemberSubscriptionOnSessionAfter
+}
+
+export interface SendEMailProps {
```

Comment:
`SendEMailProps` mostly named as `Mail` suggesting `SendMailProps`

Reply:
Yeah it still needs a bit of cleanup. there is another interface called SendMailProps.